src/Mod/PartDesign/InvoluteGearFeature.ui

```diff
@@ -173,6 +173,66 @@
      </item>
     </widget>
    </item>
+   <item row="5" column="0">
+    <widget class="QLabel" name="label">
+     <property name="text">
+      <string>Addendum Coefficient</string>
+     </property>
+    </widget>
+   </item>
+   <item row="5" column="1">
+    <widget class="QDoubleSpinBox" name="doubleSpinBox_Addendum">
+     <property name="maximum">
+      <double>5.000000000000000</double>
+     </property>
+     <property name="singleStep">
+      <double>0.050000000000000</double>
+     </property>
+     <property name="value">
+      <double>1.000000000000000</double>
+     </property>
+    </widget>
+   </item>
+   <item row="6" column="0">
+    <widget class="QLabel" name="label_2">
+     <property name="text">
+      <string>Dedendum Coefficient</string>
```

Comment:
Please also add tooltips. These will help especially newbies a lot.

Reply:
Yes, that's on my TODO list. My plan is to use the property descriptions as tool tips, in order to not maintain two times the same string. And make them translatable. Basically the same as what I've done for the Helix, cf. #5321.
I thought it would better be done as a separate PR -- separation if concerns, you know ;-)

Reply:
Fine with me.